### DISCUSSION TITLE
Copilot IntelliJ plugin has to be authenticated on each restart.

### DISCUSSION THREAD

**Discussion by busti** [community/community#7071] [Copilot Conversations] [answered]
I have installed the copilot plugin on IntelliJ and it works after logging in.  
However I need to login every time I restart IntelliJ.  
I am on Linux `Linux laptop 5.13.4 #1-NixOS SMP Tue Jul 20 14:00:42 UTC 2021 x86_64 GNU/Linux` and am using `IntelliJ IDEA  2021.2.1 (Ultimate Edition) Build #UI-212.5080.55; Runtime version: 11.0.11+0-adhoc..source amd64 VM: OpenJDK 64-Bit Server`  

If there is any information I can provide to get this issue fixed I'll happily do so.

**Comment by jansorg** [comment, marked as answer]
You're right, that's a bug. We already fixed this for the next update, which should be out soon. Thanks!
